airflow/www/webpack.config.js

```diff
@@ -102,10 +102,6 @@ const config = {
     new CleanWebpackPlugin(['static/dist']),
     new MiniCssExtractPlugin({ filename: '[name].[chunkhash].css' }),
 
-    // MomentJS loads all the locale, making it a huge JS file.
-    // This will ignore the locales from momentJS
-    new webpack.IgnorePlugin(/^\.\/locale$/, /moment$/),
```

Comment:
Do you know how much does this increase the bundle size by?

Is it worth (and possible) to make this one part a separate bundle so it can be requested and cached separately?